### PR TITLE
bodyParsers is deprecated so update with expess bodyParsers

### PR DESCRIPTION
**Pull request by cvchauhan**
- [ ] - > Body Parsers is deprecated so update with expess body Parsers
- [ ] - > Change path-to-regexp 6.2.2 to 6.3.0
- [ ] - > update express version

### CODE REVIEW

**Review by cdimascio — APPROVED** [review]

**Comment by cdimascio** [comment]
Thanks this also applies needed patch for https://github.com/cdimascio/express-openapi-validator/issues/966

**Comment by cvchauhan** [comment]
> Thanks this also applies needed patch for #966

Thanks I just forgot to modify 
examples\3-eov-operations\test.js 
app.use(bodyParser.urlencoded({ extended: false })); to app.use(express.urlencoded({ extended: false }));

**Review by cvchauhan — COMMENTED** [review]
https://github.com/cdimascio/express-openapi-validator/pull/975

File: examples/3-eov-operations/test.js [hunk 1]
```diff
@@ -10,9 +9,9 @@ const app = express();
 const apiSpec = path.join(__dirname, 'api.yaml');
 
 // 1. Install bodyParsers for the request types your API will support
-app.use(bodyParser.urlencoded({ extended: false }));
-app.use(bodyParser.text());
-app.use(bodyParser.json());
+app.use(bodyParsers.urlencoded({ extended: false }));
```
Comment (mdmower-csnw) [inline examples/3-eov-operations/test.js:12]: @cdimascio - I assume this was supposed to be `express`, not `bodyParsers`?

**Pull request merged**